app/scripts/modules/core/pipeline/config/stages/createLoadBalancer/createLoadBalancerStage.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.createLoadBalancerStage', [
+  require('../../../../utils/lodash.js'),
+  require('../../../../cloudProvider/cloudProvider.registry.js'),
+])
+  .config(function(pipelineConfigProvider, settings) {
+
+    // Register this stage only if infrastructure stages are enabled in settings.js
+    if (settings.feature.infrastructureStages) {
+      pipelineConfigProvider.registerStage({
+        key: 'upsertLoadBalancers',
+        label: 'Create Load Balancers',
+        description: 'Creates one or more load balancers',
+        templateUrl: require('./createLoadBalancerStage.html'),
+        executionDetailsUrl: require('./createLoadBalancerExecutionDetails.html'),
+        defaultTimeoutMs: 5 * 60 * 1000, // 5 minutes
+        validators: [
+        ],
+      });
+    }
+
+  })
+  .controller('createLoadBalancerStageCtrl', function($injector, $scope, $uibModal, providerSelectionService,
+                                                         cloudProviderRegistry) {
+
+    function initializeCommand() {
+      $scope.stage.loadBalancers = $scope.stage.loadBalancers || [];
+    }
+
+    this.addLoadBalancer = function() {
+      providerSelectionService.selectProvider($scope.application).then(function(selectedProvider) {
+
+        let config = cloudProviderRegistry.getValue(selectedProvider, 'loadBalancer');
+        $uibModal.open({
+          templateUrl: config.createLoadBalancerTemplateUrl,
+          controller: `${config.createLoadBalancerController} as ctrl`,
+          size: cloudProviderRegistry.getValue(selectedProvider, 'v2wizard') ? 'lg' : 'md',
+          resolve: {
+            application: function() { return $scope.application; },
+            loadBalancer: function() { return null; },
+            isNew: function() { return true; },
+            forPipelineConfig: function() { return true; }
```

Comment:
I would embrace the ES6 arrow functions for all these resolves, e.g.
```javascript
resolve: {
  application: () => $scope.application,
  loadBalancer: () => null,
  isNew: () => true,
  forPipelineConfig: () => true,
}
```
It reads a little easier. Don't hesitate to use arrow functions everywhere!